e2e/crossbrowser/supportedBrowsers.js

```diff
@@ -0,0 +1,74 @@
+const LATEST_MAC = 'macOS 10.15';
+const LATEST_WINDOWS = 'Windows 10';
+
+const supportedBrowsers = {
+  microsoft: {
+    ie11_win: {
+      browserName: 'internet explorer',
+      platformName: LATEST_WINDOWS,
+      browserVersion: 'latest',
+      'sauce:options': {
+        name: 'FPLA: IE11',
+        screenResolution: '1400x1050',
+      },
+    },
+    edge_win_latest: {
+      browserName: 'MicrosoftEdge',
+      platformName: LATEST_WINDOWS,
+      browserVersion: 'latest',
+      'sauce:options': {
```

Comment:
More of a question than a comment. IIRC, the resolution defaults to something minimal like 1024x768 when it isn't specified. Is this acceptable or expected test parameter?

Reply:
good question @imran-hmcts, so the way this config works is that it overwrites a default config defined in `saucelabs.conf.js`
```
const defaultSauceOptions = {
  username: process.env.SAUCE_USERNAME,
  accessKey: process.env.SAUCE_ACCESS_KEY,
  tunnelIdentifier: process.env.TUNNEL_IDENTIFIER || 'reformtunnel',
  acceptSslCerts: true,
  windowSize: '1600x900',
  tags: ['FPL']
};
```
So by default the window size is `1600x900` but we can override that here